base/compiler/ssair/ir.jl

```diff
@@ -679,7 +679,7 @@ function process_phinode_values(old_values::Vector{Any}, late_fixup::Vector{Int}
                 used_ssas[val.id] += 1
             end
         elseif isa(val, OldSSAValue)
-            if val.id > processed_idx
+            if val.id >= processed_idx
```

Comment:
The test passes without this one. I do think the change is correct, but if someone can come up with a piece of code that comes here with `val.id==processed_idx` to build a test case, that would be appreciated.